tox.ini

```diff
@@ -0,0 +1,269 @@
+# SPDX-License-Identifier: MIT
+[tox]
+envlist =
+    black, pylint, flake8,
+    py{26,27,36,37,38},
```

Comment:
@i386x why is 35 missing here, is it because it is the system Python version in Travis?

Reply:
yes, we do not care about this version except that it happens to be the system Python version in Travis. To be clear the pyxy environments define the unit tests.